SimDataFormats/Associations/src/classes.h

```diff
@@ -15,36 +15,52 @@
-    edm::Wrapper<reco::TrackToGenParticleAssociator> dummy2;
-    edm::Wrapper<reco::MuonToTrackingParticleAssociator> dummy3;
+struct SimDataFormats_Associations {
+  // add 'dummy' Wrapper variable for each class type you put into the Event
+  edm::Wrapper<reco::TrackToTrackingParticleAssociator> dummy1;
+  edm::Wrapper<reco::TrackToGenParticleAssociator> dummy2;
+  edm::Wrapper<reco::MuonToTrackingParticleAssociator> dummy3;
 
-    edm::Wrapper<reco::VertexToTrackingVertexAssociator> dummy4;
+  edm::Wrapper<reco::VertexToTrackingVertexAssociator> dummy4;
 
-    edm::Wrapper<hgcal::LayerClusterToCaloParticleAssociator> dummy5;
+  edm::Wrapper<hgcal::LayerClusterToCaloParticleAssociator> dummy5;
 
-    edm::Wrapper<hgcal::LayerClusterToSimClusterAssociator> dummy6;
+  edm::Wrapper<hgcal::LayerClusterToSimClusterAssociator> dummy6;
 
-    edm::Wrapper<hgcal::TracksterToSimClusterAssociator> dummy7;
+  edm::Wrapper<hgcal::TracksterToSimClusterAssociator> dummy7;
 
-    edm::Wrapper<hgcal::MultiClusterToCaloParticleAssociator> dummy8;
+  edm::Wrapper<hgcal::MultiClusterToCaloParticleAssociator> dummy8;
 
-    edm::Wrapper<hgcal::TracksterToSimTracksterAssociator> dummy9;
+  edm::Wrapper<hgcal::TracksterToSimTracksterAssociator> dummy9;
 
-    edm::Wrapper<hgcal::TracksterToSimTracksterHitLCAssociator> dummy10;
+  edm::Wrapper<hgcal::TracksterToSimTracksterHitLCAssociator> dummy10;
 
-    edm::Wrapper<hgcal::LayerClusterToSimTracksterAssociator> dummy11;
+  edm::Wrapper<hgcal::LayerClusterToSimTracksterAssociator> dummy11;
 
-    reco::VertexSimToRecoCollection vstrc;
-    reco::VertexSimToRecoCollection::const_iterator vstrci;
-    edm::Wrapper<reco::VertexSimToRecoCollection> wvstrc;
+  edm::Wrapper<reco::MtdRecoClusterToSimLayerClusterAssociator> dummy12;
 
-    reco::VertexRecoToSimCollection vrtsc;
-    reco::VertexRecoToSimCollection::const_iterator vrtsci;
-    edm::Wrapper<reco::VertexRecoToSimCollection> wvrtsci;
-  };
-}  // namespace SimDataFormats_Associations
+  reco::VertexSimToRecoCollection vstrc;
+  reco::VertexSimToRecoCollection::const_iterator vstrci;
+  edm::Wrapper<reco::VertexSimToRecoCollection> wvstrc;
+
+  reco::VertexRecoToSimCollection vrtsc;
+  reco::VertexRecoToSimCollection::const_iterator vrtsci;
+  edm::Wrapper<reco::VertexRecoToSimCollection> wvrtsci;
+
+  std::pair<FTLClusterRef, std::vector<MtdSimLayerClusterRef>> dummy13;
+  edm::Wrapper<std::pair<FTLClusterRef, std::vector<MtdSimLayerClusterRef>>> dummy14;
+  MtdRecoClusterToSimLayerClusterAssociationMap dummy15;
+  edm::Wrapper<MtdRecoClusterToSimLayerClusterAssociationMap> dummy16;
+
+  std::pair<MtdSimLayerClusterRef, std::vector<FTLClusterRef>> dummy17;
+  edm::Wrapper<std::pair<MtdSimLayerClusterRef, std::vector<FTLClusterRef>>> dummy18;
+  MtdSimLayerClusterToRecoClusterAssociationMap dummy19;
+  edm::Wrapper<MtdSimLayerClusterToRecoClusterAssociationMap> dummy20;
+
+  edm::Wrapper<reco::MtdSimLayerClusterToTPAssociator> dummy21;
+};
```

Comment:
This `struct SimDataFormats_Associations { ... };` could be removed (those have been unnecessary for a long time), but at this stage it could be left for a future PR.

Reply:
@makortel do you mean the whole block lines 23 - 66? We may add small PR after this one is integrated

Reply:
> do you mean the whole block lines 23 - 66?

Yes.

> We may add small PR after this one is integrated

Thanks!